manifesto.md

```diff
@@ -71,10 +71,6 @@ Discussions and decisions are open, asynchronous and archived.
 
 Results of commits can be seen in a production environment in 60 seconds or less.
 
```

Comment:
```suggestion
### MVP Done Right

It's "minimum" when the team has doubts that it could work. It's viable when users don't care and use it anyway.
```

Maybe a more accurate description of our approach to MVP. We don't declare MVP because we have run out of time and need to ship, we ship something that isn't proven and iterate on the feedback.

Reply:
A bit cryptic, but ok... 😄